modules/tinymce/src/plugins/media/test/ts/browser/DataToHtmlTest.ts

```diff
@@ -71,4 +72,86 @@ describe('browser.tinymce.plugins.media.core.DataToHtmlTest', () => {
     },
     iframeStruct
   ));
+
+  it('TINY-8684: Assert html structure of audio from template callback', async () => {
```

Comment:
This should not have been changed and should have remained as `TBA` as only the iframe callback applies to this.

- [ ] Revert `TBA` test ids for the video/audio tests